enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -425,15 +430,30 @@ func TestFetchRemoteBuildOutputs(t *testing.T) {
 	cmd.Stdout = &buf
 	err = cmd.Run()
 	require.NoError(t, err)
-	require.Equal(t, "Hello! I'm a go program.\n", buf.String())
+	require.Equal(t, "Hello from main!", buf.String())
 }
 
 func TestBuildRemotelyRunLocally(t *testing.T) {
-	repoDir := clonePrivateTestRepo(t)
+	repoDir, _ := makeLocalGitRepo(t, map[string]string{
+		"BUILD": `
+cc_binary(
+    name = "main",
+    srcs = ["main.c"],
+)
+`,
+		"main.c": `
+#include <stdio.h>
+
+int main() {
+    printf("Hello from main!");
+    return 0;
+}
+`,
+		"WORKSPACE": "",
+	})
```

Comment:
is there something specific we're testing by using cc_binary? wondering if `genrule` would be good enough, since the `testbazel` binary pre-caches all of the dependencies needed for genrule to work.

Reply:
Yeah I wanted to test building a binary on a remote runner, fetching it, and running it locally. We have some complicated logic related to parsing [runfile events ](https://github.com/buildbuddy-io/buildbuddy/blob/master/cli/remotebazel/remotebazel.go#L951) that I wanted coverage for, and I couldn't figure out how to test that with `genrule` or `sh_binary`. To be honest I'm not very familiar with those events though, so if you think it's possible to accomplish this with a genrule please let me know

Reply:
Going to merge for now, but can update the tests if there's a simpler and faster way to do this